wasm2c/examples/fac/fac.h

```diff
@@ -29,9 +29,9 @@ typedef struct Z_fac_instance_t {
   char dummy_member;
 } Z_fac_instance_t;
 
-void Z_fac_init_module(void);
 void Z_fac_instantiate(Z_fac_instance_t*);
 void Z_fac_free(Z_fac_instance_t*);
+wasm_rt_func_type_t Z_fac_get_func_type(uint32_t param_count, uint32_t result_count, ...);
```

Comment:
Does this need to be a public function?

Reply:
I think so -- the purpose is to help the host generate a funcref that can be passed to Wasm code (as in the new "callback" example). The function isn't called by any of the wasm2c-generated code.